pkg/cmd/roachtest/tests/multitenant_upgrade.go

```diff
@@ -82,8 +81,11 @@ func runMultiTenantUpgrade(
 	currentBinaryMinSupportedVersion, ok := versionToMinSupportedVersion[curBinaryMajorAndMinorVersion]
 	require.True(t, ok, "current binary '%s' not found in 'versionToMinSupportedVersion' map", curBinaryMajorAndMinorVersion)
 
-	predecessor, err := release.LatestPredecessor(v)
-	require.NoError(t, err)
+	// predecessor, err := release.LatestPredecessor(v)
+	// require.NoError(t, err)
+	// Hard-code the pre-decessor release to 23.1.4 until a new patch release is out (23.1.9) because
+	// the test is in-compatible with 23.1.{5,6,7,8} due to an erroneous PR merged on the 23.1 branch.
+	predecessor := "23.1.4"
```

Comment:
Thanks. Could you make this override temporary by actually encoding a condition on the output of `release.LatestPredecessor`? Something like:

```
if major == 23 and minor == 1 and patch < 7 {
       predecessor = "23.1.4"
}
```

so that the condition automatically ceases to apply once .7 is released.

Reply:
Done